readme.md

```diff
@@ -46,10 +46,16 @@ Annotation processing has quite a few advantages over external tools or build to
 - Can "react" to annotations provided by the framework.
 - Annotation processing is performed in rounds which makes it technically easier to write extensions and customizations.
 
-#### What this project will not do
-This project will not trigger container builds of any kind (e.g. docker, openshift builds), nor will it communicate with any daemon or api server.
-Integration with external tools that perform such task might be added in the future.
+#### What about building and deploying?
+Performing container builds like docker, or openshift source to image builds, is outside of the scope of this project.
```

Comment:
This sentence should be rephrased to something like. `Triggering the build of a docker image using a Dockerfile or a S2i image is outside of the scope of this project but nevertheless, it is proposed as an experimental option using of the following hooks till we have figured out how such deployment/integration will take place with different tools such as : fabric8 maven plugin, odo, launcher, ... or been delegated to an operator able to process the resources generated`